tutorials/userns.md

```diff
@@ -0,0 +1,65 @@
+# User namespaces and CRI-O
+
+The purpose of this document is to show how to configure user namespaces in CRI-O, as well as some of the options CRI-O supports for configuring user namespaces.
+
+## Setup
+
+### /etc/sub{g,u}id
+To start, the host will have to have `/etc/subuid` and `/etc/subgid` files set correctly. By default, the [library CRI-O uses for container storage](https://github.com/containers/storage) assumes there will be entries in each of these files for the `containers` user. If one would like to have a different user's entries in `/etc/sub?id` files, then the field `remap-user` and `remap-group` can be configured in `/etc/containers/storage.conf` in the `[storage.options]` table.
```

Comment:
I try to avoid pronouns on a man page.  Suggest dropping "If one would like" at the start of the third sentence for instance.  Etc throughout, YMMV.